elastic/kibana
# [AO] Update the alert details page title

Closes #152595

## Summary

This PR updates the alert details page title. The title will be as follows:

<image src="https://user-images.githubusercontent.com/12370520/229131794-51604a72-9d04-44a4-8bce-e2a520844c0b.png" width="300" />

Example of how it looks like now:
![image](https://user-images.githubusercontent.com/12370520/229133453-9e2d2ae8-109f-4859-a797-5b577fe7ee1d.png)

I used `kibana.alert.rule.category` + breached/detected as the title.

You can also check this component in the storybook.


https://user-images.githubusercontent.com/12370520/229860422-274e3ae4-38f0-4ef5-82f3-b11ccf21729a.mov


## Questions
1. If we use the alert name as a variable in the page title, will the translation work correctly? 
    Yes, if we use `select` in translation, we can set different verbs for each rule type.
2. Is there any considerations in changing the alert name? Do I need to remove the related translation?
    Since changing the name will not be applied to the previous alert documents, I will not change that in this PR.

## 🧪 How to test
1. Add `xpack.observability.unsafe.alertDetails.metrics.enabled: true` to the Kibana config
2. Generate metric threshold alert
3. Go to the related alert details page and check the title
